DiscordChatExporter.Core/Exporting/MessageExporter.cs

```diff
@@ -30,6 +30,11 @@ private async ValueTask ResetWriterAsync(CancellationToken cancellationToken = d
         }
     }
 
+    // Unless explicitly called, file will be created before writing first message
+    public async ValueTask EnsureFileIsCreated(CancellationToken cancellationToken = default) {
+        await GetWriterAsync(cancellationToken);
+    }
```

Comment:
Might be more explicit than strictly needed, but it documents intended behavior.